### ISSUE TITLE
A set of Lenovo A850 devices constantly drop connections 

### ISSUE THREAD

**Issue by SergKhachko** [openstf/stf#283]
STFService successfully installed and running (all device information is available on Devices list), but it's impossible to remote control the device (the device physically disconnects with Try Again message in STF dashboard).
Tried to use another USB ports/hubs with no luck. USB debug mode configured successfully too.
Log file is attached.
[a850.zip](https://github.com/openstf/stf/files/177616/a850.zip)

Please, help.

**Comment by sorccu** [comment]
Yeah some Lenovo devices are known not to work with STF right now. Personally I don't have any so I can't really do anything. Please consider donating the problematic Lenovo device or a similar device to us.

Alternatively, you can click the badge below to start a bounty on this issue and maybe we or someone else will take a look at it if it reaches a reasonable value.

[![Bountysource](https://api.bountysource.com/badge/issue?issue_id=31981649)](https://www.bountysource.com/issues/31981649-a-set-of-lenovo-a850-devices-constantly-drop-connections)

**Comment by sorccu** [comment]
There's a good chance that your device might work now in STF 2.3.0. Please test and report back!
